src/Glpi/Form/QuestionType/AbstractQuestionTypeShortAnswer.php

```diff
@@ -134,12 +135,21 @@ class="form-control"
     public function renderEndUserTemplate(
         Question $question,
     ): string {
+        $defaul_value = $question->fields['default_value'] ?? '';
```

Comment:
There is a typo even on twig variable name.
```suggestion
        $default_value = $question->fields['default_value'] ?? '';
```